src/assets/styles.scss

```diff
@@ -0,0 +1,498 @@
+@import "variables.scss";
+/* hub-specific overrides to variables */
+$font-family-sans-serif: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell",
+    "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+
+$headings-font-family: $font-family-sans-serif;
+$headings-font-weight: 300;
+
+$font-family-base: $font-family-sans-serif;
+
+$font-size-base: 0.875rem;
+$line-height-base: 1.6;
+$headings-line-height: 1.6;
+
+$h1-font-size: $font-size-base * 2; // default 2.5
+$h2-font-size: $font-size-base * 1.75; // default 2
+$h3-font-size: $font-size-base * 1.5; // default 1.75
+$h4-font-size: $font-size-base * 1.25; // default 1.5
+$h5-font-size: $font-size-base; // default 1.25
+$h6-font-size: $font-size-base * 0.85; // default 1
+$blockquote-font-size: $font-size-base;
+
+$brand-primary: #336699;
+$headings-small-color: inherit;
+
+$body-bg: white;
+$navbar-default-bg: rgb(255, 255, 255);
+$navbar-default-border: none;
+
+$navbar-inverse-bg: rgb(44, 49, 67);
+
+$grid-float-breakpoint: 1000px;
+
+// splashpage carousel styling
+$jumbotron-bg: white;
+$carousel-indicator-border-color: gray;
+$carousel-indicator-active-bg: $navbar-inverse-bg;
+$carousel-control-width: 5%;
+
+@import "~bootstrap/scss/bootstrap.scss";
+
+//@import "~@fortawesome/fontawesome-free/scss/_variables.scss";
+$fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
+@import "~@fortawesome/fontawesome-free/scss/fontawesome.scss";
```

Comment:
This import is failing in the deployment via netlify: https://app.netlify.com/sites/hungry-aryabhata-193bcf/deploys/61fcf438507d8d00080a5ace
You should add `fortawesome` in the `package.json`: see https://github.com/galaxyproject/galaxy-hub/blob/7fff11d849102f18e2ab5b0a4e07d5881957ff8a/package.json#L22